client/components/tinymce/plugins/wpcom/plugin.js

```diff
@@ -32,7 +32,7 @@ function wpcomPlugin( editor ) {
 
 		if ( event.content ) {
 			if ( event.content.indexOf( '<!--more' ) !== -1 ) {
-				title = __( 'Read more...' );
```

Comment:
This should be removed from `client/components/tinymce/i18n` if we're switching to `i18n-calypso`.